apps/music/js/ui/views/player_view.js

```diff
@@ -228,19 +230,17 @@ var PlayerView = {
         this.album.classList.remove('hidden-cover-share');
       }
     } else {
-      var titleBar = document.getElementById('title-text');
-
-      titleBar.textContent =
-        metadata.title || navigator.mozL10n.get('unknownTitle');
-      titleBar.dataset.l10nId = metadata.title ? '' : 'unknownTitle';
+      TitleBar.changeTitleText(
```

Comment:
Also you used `TitleBar.titleText` in the else case in the last commit, note that it will break the open activity as well so let's keep the original approach.